src/index.tsx

```diff
@@ -2,12 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
+import log from 'loglevel';
 
 import './styles.scss';
 import App from './components/App';
 import configureStore from './configureStore';
 import { actions as apiActions } from './reducers/api';
 
+if (process.env.NODE_ENV === 'production') {
+  log.setLevel(log.levels.INFO);
+}
```

Comment:
Can you add an `else` branch that configures debug logging? The docs don't say what the default level will be so I think it's better to make it explicit.